medusa/name_parser/rules/rules.py

```diff
@@ -35,8 +35,8 @@
 
 from guessit.rules.common.comparators import marker_sorted
 from guessit.rules.common.formatters import cleanup
-from guessit.rules.properties import website
-from guessit.rules.properties.release_group import clean_groupname
+# @FIXME: `clean_groupname` is no longer available...
+# from guessit.rules.properties.release_group import clean_groupname
```

Comment:
It was a public function, now it's an inner function inside the `release_group` function.
`clean_groupname` is used in `ReleaseGroupPostProcessor`.